docs/Working Agreement.md

```diff
@@ -0,0 +1,163 @@
+# Working Agreement
```

Comment:
Is this meant to include the changes and suggestions in Ayo's PR?  Or is it meant to be out of date until his PR is done?